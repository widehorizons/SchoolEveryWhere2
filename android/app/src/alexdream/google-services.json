{
  "project_info": {
    "project_number": "224420084449",
    "firebase_url": "https://alex-dream.firebaseio.com",
    "project_id": "alex-dream",
    "storage_bucket": "alex-dream.appspot.com"
  },
  "client": [
    {
      "client_info": {
        "mobilesdk_app_id": "1:224420084449:android:0f3dfcfb06aa499859d5a0",
        "android_client_info": {
          "package_name": "com.schooleverywhere.alexdream"
        }
      },
      "oauth_client": [
        {
          "client_id": "224420084449-8imvobt62eiv44r0eln4quf62q0oq09o.apps.googleusercontent.com",
          "client_type": 1,
          "android_info": {
            "package_name": "com.schooleverywhere.alexdream",
            "certificate_hash": "ccc1299bc2b52e07f4dc544ce118a49c5b3db074"
          }
        },
        {
          "client_id": "224420084449-nq7sk132b5o9rvrr6mredj7k9stq4u5s.apps.googleusercontent.com",
          "client_type": 1,
          "android_info": {
            "package_name": "com.schooleverywhere.alexdream",
            "certificate_hash": "83d93cea0713324423466838ddad429ef2673f66"
          }
        },
        {
          "client_id": "224420084449-vrbchug6mt0obdak6vdc15mfrfl0tde4.apps.googleusercontent.com",
          "client_type": 3
        }
      ],
      "api_key": [
        {
          "current_key": "AIzaSyCM48FjXaOj4cJcvBsG8sbVBigRMbjBlxA"
        }
      ],
      "services": {
        "appinvite_service": {
          "other_platform_oauth_client": [
            {
              "client_id": "224420084449-vrbchug6mt0obdak6vdc15mfrfl0tde4.apps.googleusercontent.com",
              "client_type": 3
            }
          ]
        }
      }
    }
  ],
  "configuration_version": "1"
}