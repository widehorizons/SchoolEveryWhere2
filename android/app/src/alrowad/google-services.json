{
  "project_info": {
    "project_number": "727402805038",
    "project_id": "alrowad-71e1b",
    "storage_bucket": "alrowad-71e1b.appspot.com"
  },
  "client": [
    {
      "client_info": {
        "mobilesdk_app_id": "1:727402805038:android:d0a61e4e25559a74bb6cbf",
        "android_client_info": {
          "package_name": "com.schooleverywhere.alrowad"
        }
      },
      "oauth_client": [
        {
          "client_id": "727402805038-6rd8c960rm5nheejl1ludcpdq54eqg2s.apps.googleusercontent.com",
          "client_type": 1,
          "android_info": {
            "package_name": "com.schooleverywhere.alrowad",
            "certificate_hash": "cf7fd48420506b1b728e1f0c17b29763c0f15e22"
          }
        },
        {
          "client_id": "727402805038-q72nu1ti19rapfnkgu91e64qoobm7vg5.apps.googleusercontent.com",
          "client_type": 3
        }
      ],
      "api_key": [
        {
          "current_key": "AIzaSyDLD1sT_emSE0SSjmytnG9Jx2voIHLILLA"
        }
      ],
      "services": {
        "appinvite_service": {
          "other_platform_oauth_client": [
            {
              "client_id": "727402805038-q72nu1ti19rapfnkgu91e64qoobm7vg5.apps.googleusercontent.com",
              "client_type": 3
            }
          ]
        }
      }
    }
  ],
  "configuration_version": "1"
}