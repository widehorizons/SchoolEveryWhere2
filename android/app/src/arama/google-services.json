{
  "project_info": {
    "project_number": "458133990886",
    "project_id": "arama-e9f30",
    "storage_bucket": "arama-e9f30.appspot.com"
  },
  "client": [
    {
      "client_info": {
        "mobilesdk_app_id": "1:458133990886:android:13af379ac5fe91afa87a27",
        "android_client_info": {
          "package_name": "com.schooleverywhere.arama"
        }
      },
      "oauth_client": [
        {
          "client_id": "458133990886-f97reb3e689rrool5amdfa0qpebp8ib8.apps.googleusercontent.com",
          "client_type": 3
        }
      ],
      "api_key": [
        {
          "current_key": "AIzaSyCEr43gFCoRDR_nkBSc0UEdaauDUnv4nXQ"
        }
      ],
      "services": {
        "appinvite_service": {
          "other_platform_oauth_client": [
            {
              "client_id": "458133990886-f97reb3e689rrool5amdfa0qpebp8ib8.apps.googleusercontent.com",
              "client_type": 3
            }
          ]
        }
      }
    }
  ],
  "configuration_version": "1"
}