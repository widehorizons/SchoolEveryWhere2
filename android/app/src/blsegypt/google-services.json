{
  "project_info": {
    "project_number": "394998152723",
    "project_id": "british-language-school",
    "storage_bucket": "british-language-school.appspot.com"
  },
  "client": [
    {
      "client_info": {
        "mobilesdk_app_id": "1:394998152723:android:0be25536673a2636c609e6",
        "android_client_info": {
          "package_name": "com.schooleverywhere.blsegypt"
        }
      },
      "oauth_client": [
        {
          "client_id": "394998152723-g40pbudebvfm0f2dh3c990o4fbjaousg.apps.googleusercontent.com",
          "client_type": 3
        }
      ],
      "api_key": [
        {
          "current_key": "AIzaSyChu-uuewrGqT5tTwgQayOPKuAn-G-6Hdo"
        }
      ],
      "services": {
        "appinvite_service": {
          "other_platform_oauth_client": [
            {
              "client_id": "394998152723-g40pbudebvfm0f2dh3c990o4fbjaousg.apps.googleusercontent.com",
              "client_type": 3
            }
          ]
        }
      }
    }
  ],
  "configuration_version": "1"
}