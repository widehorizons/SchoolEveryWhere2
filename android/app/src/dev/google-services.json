{
    "project_info": {
        "project_number": "839579222585",
        "firebase_url": "https://try-schooleverywhere.firebaseio.com",
        "project_id": "try-schooleverywhere",
        "storage_bucket": "try-schooleverywhere.appspot.com"
    },
    "client": [{
        "client_info": {
            "mobilesdk_app_id": "1:839579222585:android:2e2634d648a66e1fff83e8",
            "android_client_info": {
                "package_name": "com.schooleverywhere.schooleverywhere"
            }
        },
        "oauth_client": [{
                "client_id": "839579222585-mnn6e0sbdapbrlg7bqsdj3s0ojdecgo8.apps.googleusercontent.com",
                "client_type": 1,
                "android_info": {
                    "package_name": "com.schooleverywhere.schooleverywhere",
                    "certificate_hash": "e1809fbe8a1718b7e85bca7cc04a31b6da603b50"
                }
            },
            {
                "client_id": "839579222585-5grh460pf6r8eq6p7mm29cegc8c93r0h.apps.googleusercontent.com",
                "client_type": 3
            }
        ],
        "api_key": [{
            "current_key": "AIzaSyB95ufs2P3NT7PBmLkz9mRJ-i0d5JdPdh8"
        }],
        "services": {
            "appinvite_service": {
                "other_platform_oauth_client": [{
                    "client_id": "839579222585-5grh460pf6r8eq6p7mm29cegc8c93r0h.apps.googleusercontent.com",
                    "client_type": 3
                }]
            }
        }
    }],
    "configuration_version": "1"
}