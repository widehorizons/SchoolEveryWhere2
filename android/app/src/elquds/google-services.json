{
  "project_info": {
    "project_number": "502503902742",
    "firebase_url": "https://elquds-5753c.firebaseio.com",
    "project_id": "elquds-5753c",
    "storage_bucket": "elquds-5753c.appspot.com"
  },
  "client": [
    {
      "client_info": {
        "mobilesdk_app_id": "1:502503902742:android:7d7fa9bf6c9ea26d882f37",
        "android_client_info": {
          "package_name": "com.elquds.schooleverywhere"
        }
      },
      "oauth_client": [
        {
          "client_id": "502503902742-dvto0d75ek18gjd8ch8gn414hhf6hrbr.apps.googleusercontent.com",
          "client_type": 1,
          "android_info": {
            "package_name": "com.elquds.schooleverywhere",
            "certificate_hash": "ccc1299bc2b52e07f4dc544ce118a49c5b3db074"
          }
        },
        {
          "client_id": "502503902742-86sgs1h4al21r1hgl4um5rs433lnnqmm.apps.googleusercontent.com",
          "client_type": 3
        }
      ],
      "api_key": [
        {
          "current_key": "AIzaSyDVgWYkpuYHBFfL3jAO40Q8j2DSLPXP_XI"
        }
      ],
      "services": {
        "appinvite_service": {
          "other_platform_oauth_client": [
            {
              "client_id": "502503902742-86sgs1h4al21r1hgl4um5rs433lnnqmm.apps.googleusercontent.com",
              "client_type": 3
            }
          ]
        }
      }
    }
  ],
  "configuration_version": "1"
}