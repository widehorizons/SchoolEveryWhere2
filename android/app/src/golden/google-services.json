{
  "project_info": {
    "project_number": "815833167235",
    "project_id": "golden-school-98c33",
    "storage_bucket": "golden-school-98c33.appspot.com"
  },
  "client": [
    {
      "client_info": {
        "mobilesdk_app_id": "1:815833167235:android:820ac9540a6b99c0ab72b6",
        "android_client_info": {
          "package_name": "com.schooleverywhere.golden"
        }
      },
      "oauth_client": [
        {
          "client_id": "815833167235-v85a6bp9710s2f5m82lk7qjg48rj3k9p.apps.googleusercontent.com",
          "client_type": 3
        }
      ],
      "api_key": [
        {
          "current_key": "AIzaSyCg8B_wbYYxjAV_nAJBQFoYq330U1LXe-0"
        }
      ],
      "services": {
        "appinvite_service": {
          "other_platform_oauth_client": [
            {
              "client_id": "815833167235-v85a6bp9710s2f5m82lk7qjg48rj3k9p.apps.googleusercontent.com",
              "client_type": 3
            }
          ]
        }
      }
    }
  ],
  "configuration_version": "1"
}